src/openeo.js

```diff
@@ -26,7 +26,7 @@ const Parameter = require('./builder/parameter');
 const Formula = require('./builder/formula');
 
 const MIN_API_VERSION = '1.0.0-rc.2';
-const MAX_API_VERSION = '1';
+const MAX_API_VERSION = '1.0.x';
```

Comment:
Thank you!

```suggestion
const MAX_API_VERSION = '1.x.x';
```
This can actually allow all versions starting with 1, which was my understanding of using "1", too. But obviously that was not correct.

Reply:
Even better, but then the README needs to be updated ;-)

Reply:
Indeed, will update the README, too. Updates for the JS client and Web Editor will be rolled out in the next hours.

Reply:
Cool. Thx.